@xen-orchestra/lite/src/stories/web-core/ui/circle-progress-bar/ui-circle-progress-bar.story.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('value').num().preset(75).required().widget(),
+      prop('maxValue').num().preset(100).widget(),
+      prop('accent').required().enum('info', 'warning', 'danger').preset('info').widget(),
```

Comment:
To have complete documentation, you could add this code in a new file called `ui-circle-progress-bar.story.md`:
```ts
type ProgressCircleAccent = 'info' | 'warning' | 'danger'
type ProgressCircleSize = 'extra-small' | 'small' | 'medium' | 'large'
```
And one thing I didn't notice at first was that the types’ names should match the component's name, so:
```ts
type CircleProgressBarAccent = 'info' | 'warning' | 'danger'
type CircleProgressBarSize = 'extra-small' | 'small' | 'medium' | 'large'
```